docs/api-reference/directives/deprecated.mdx

```diff
@@ -0,0 +1,30 @@
+---
+id: deprecated
+title: deprecated
+---
+
+export const Tag = ({children, color}) => (
+<span
+style={{
+      backgroundColor: color,
+      borderRadius: '2px',
+      color: '#fff',
+      padding: '0.2rem',
+    }}>
+{children}
+</span>
+);
```

Comment:
What's that bit? It seems to be duplicated for each individual page, can we replace this with an import? Should this use theme colors?

Reply:
![image](https://user-images.githubusercontent.com/5188636/105512365-4f1aaa80-5cd1-11eb-8155-1319e07f0a90.png)
Thats the bit responsible for rendering deprecated badge. Will migrate it to common stylesheet 👍

Reply:
@patrys fixed